ci/cloudbuild/builds/cmake-oldest-deps.sh

```diff
@@ -36,6 +36,18 @@ io::log_h2 "Building"
 cmake --build cmake-out/build
 
 io::log_h2 "Testing"
-env -C cmake-out/build ctest -LE "integration-test" --parallel "$(nproc)"
+
+ctest_flags=(
```

Comment:
Perhaps `s/ctest_flags/ctest_args/` to match the others files?

Reply:
Done